Analysis/Tasks/PWGDQ/dileptonEE.cxx

```diff
@@ -283,86 +401,41 @@ void DefineHistograms(o2::framework::OutputObj<HistogramManager> histMan, TStrin
-                            "", "", "", VarManager::kNothing, VarManager::kVtxNcontrib); // TH1F histogram, filled with weights using the vtx n-contributors
-
-      Int_t vars[4] = {VarManager::kVtxX, VarManager::kVtxY, VarManager::kVtxZ, VarManager::kVtxNcontrib};
-      TArrayD binLimits[4];
-      binLimits[0] = TArrayD(10, vtxXbinLims);
-      binLimits[1] = TArrayD(7, vtxYbinLims);
-      binLimits[2] = TArrayD(13, vtxZbinLims);
-      binLimits[3] = TArrayD(9, nContribbinLims);
-      histMan->AddHistogram(classStr.Data(), "vtxHisto", "n contrib vs (x,y,z)", 4, vars, binLimits);
-
-      histMan->AddHistogram(classStr.Data(), "CentV0M_vtxZ", "CentV0M vs Vtx Z", false, 60, -15.0, 15.0, VarManager::kVtxZ, 20, 0., 100., VarManager::kCentVZERO); // TH2F histogram
-
-      histMan->AddHistogram(classStr.Data(), "VtxChi2", "Vtx chi2", false, 100, 0.0, 100.0, VarManager::kVtxChi2); // TH1F histogram
-
+      histMan->AddHistogram(classStr.Data(), "VtxZ", "Vtx Z", false, 60, -15.0, 15.0, VarManager::kVtxZ);                                                           // TH1F histogram
+      histMan->AddHistogram(classStr.Data(), "CentV0M_vtxZ", "CentV0M vs Vtx Z", false, 60, -15.0, 15.0, VarManager::kVtxZ, 100, 0., 100., VarManager::kCentVZERO); // TH2F histogram
       continue;
     } // end if(Event)
 
     if (classStr.Contains("Track")) {
       histMan->AddHistClass(classStr.Data());
-      histMan->AddHistogram(classStr.Data(), "Pt", "p_{T} distribution", false, 100, 0.0, 10.0, VarManager::kPt);                                                       // TH1F histogram
-      histMan->AddHistogram(classStr.Data(), "Eta", "#eta distribution", false, 200, -1.0, 1.0, VarManager::kEta);                                                      // TH1F histogram
-      histMan->AddHistogram(classStr.Data(), "Phi_Eta", "#phi vs #eta distribution", false, 200, -1.0, 1.0, VarManager::kEta, 72, 0, TMath::TwoPi(), VarManager::kPhi); // TH2F histogram
-      histMan->AddHistogram(classStr.Data(), "P", "p distribution", false, 200, 0.0, 20.0, VarManager::kP);                                                             // TH1F histogram
-      histMan->AddHistogram(classStr.Data(), "Px", "p_{x} distribution", false, 200, 0.0, 20.0, VarManager::kPx);
-      histMan->AddHistogram(classStr.Data(), "Py", "p_{y} distribution", false, 200, 0.0, 20.0, VarManager::kPy);
-      histMan->AddHistogram(classStr.Data(), "Pz", "p_{z} distribution", false, 400, -20.0, 20.0, VarManager::kPz);
-      histMan->AddHistogram(classStr.Data(), "DCAxy_DCAz", "DCA_{xy} vs DCA_{z}", false, 100, -5.0, 5.0, VarManager::kTrackDCAxy, 100, -5.0, 5.0, VarManager::kTrackDCAz); // TH2F histogram
+      histMan->AddHistogram(classStr.Data(), "Pt", "p_{T} distribution", false, 200, 0.0, 20.0, VarManager::kPt);                                                // TH1F histogram
```

Comment:
you can use the histogram library, most of these histograms should be defined there already